src/test/java/teammates/test/pageobjects/InstructorFeedbackSessionsPage.java

```diff
@@ -255,17 +256,19 @@ public void addFeedbackSessionWithStandardTimeZone(String feedbackSessionName, S
             LocalDateTime startTime, LocalDateTime endTime, LocalDateTime visibleTime, LocalDateTime publishTime,
             Text instructions, int gracePeriod) {
 
-        addFeedbackSessionWithTimeZone(
-                feedbackSessionName, courseId, startTime, endTime, visibleTime, publishTime, instructions, gracePeriod, 8.0);
+        addFeedbackSessionWithTimeZone(feedbackSessionName, courseId, startTime, endTime, visibleTime, publishTime,
+                instructions, gracePeriod, TimeHelper.convertToZoneId(8));
```

Comment:
Just use something like `ZoneId.of("+08:00")`? `convertToZoneId` should be removed eventually.